src/EA.Weee.Web/EA.Weee.Web.csproj

```diff
@@ -902,6 +901,8 @@
     <Content Include="Areas\Admin\Views\EvidenceTransferReport\EvidenceTransferNoteReport.cshtml" />
     <Content Include="Areas\Scheme\Views\Shared\EditorTemplates\RecipientStatusFiltersViewModel.cshtml" />
     <Content Include="Areas\Scheme\Views\Shared\_NoResultsPartial.cshtml" />
+    <Content Include="Areas\Aatf\Views\Shared\EditorTemplates\FilterViewModel.cshtml" />
```

Comment:
@NateHogan Check if its possible to reuse these views rather than duplicating...